internal/commands/helpers.go

```diff
@@ -9,11 +9,11 @@ import (
 func getStorageProvider() (provider storage.Provider, err error) {
```

Comment:
```suggestion
func getStorageProvider() (provider storage.Provider) {
```

This makes the func more universally useful.